rootfsCloner/rootfsCloner.go

```diff
@@ -0,0 +1,281 @@
+//
+// Copyright 2022 Nestybox, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package rootfsCloner
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/nestybox/sysbox-libs/formatter"
+	"github.com/opencontainers/runc/libcontainer/mount"
```

Comment:
Why don't you use sysbox-runc's equivalent package instead? Unless we're using a recent change from oci-runc (not present in sysbox-runc yet), there shouldn't be any runc dependency from sysbox-mgr (or any other sysbox component) -- the exception being sysbox-runc itself, of course.

Reply:
Good point, fixed.